JavaFiddle-ejb/src/main/java/Test.java

```diff
@@ -0,0 +1,67 @@
+
+
+import ru.javafiddle.core.ejb.ProjectBean;
+import ru.javafiddle.jpa.entity.*;
+import ru.javafiddle.core.ejb.UserBean;
+import ru.javafiddle.core.ejb.GroupBean;
+
+import javax.faces.event.SystemEvent;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import java.io.UnsupportedEncodingException;
+import java.security.NoSuchAlgorithmException;
+
+
+/**
+ * Created by artyom on 06.02.16.
+ */
+public class Test {
```

Comment:
This class was removed by Artem after code review as making no value.
Could you please perform a correct rebase on Development removing all unnecessary code from your Pull Request?